### PR TITLE
update index.d.ts

### PR DESCRIPTION
**Pull request by ORESoftware**
the constructor takes an optional string first argument, and an optional options second argument

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by ORESoftware** [comment]
this PR relates to this issue: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/25439

the relevant code for the mongodb v3 library is here:
https://github.com/mongodb/node-mongodb-native/blob/master/lib/mongo_client.js#L261

File: types/mongodb/index.d.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export { Binary, Double, Long, Decimal128, MaxKey, MinKey, ObjectID, ObjectId, T
 
 // Class documentation : http://mongodb.github.io/node-mongodb-native/3.0/api/MongoClient.html
 export class MongoClient extends EventEmitter {
-    constructor();
+    constructor(uri?: string, options?: MongoClientOptions);
```
Comment (CaselIT) [inline types/mongodb/index.d.ts]: From the documentation, the URL argument is required

**Review by CaselIT — CHANGES_REQUESTED** [review]
Thanks for your contribution

File: types/mongodb/index.d.ts (hunk 1 above)
Comment (ORESoftware) [reply types/mongodb/index.d.ts]: ok, I will update it, looks like you're right

**Review by CaselIT — APPROVED** [review]
It's ok now. Thanks

**Comment by ORESoftware** [comment]
Sorry I am a newb - once a PR is merged for typings, is it available via NPM immediately, or does it get published on an interval? I was thinking that maybe the NPM package for `@types/x` actually pulled from Github via the preinstall/postinstall hooks.

**Comment by andy-ms** [comment]
@ORESoftware It looks like a new version 3.0.17 has been published. The npm package doesn't automatically pull from github, we have a process to publish from DT to npm that may take a few hours.

**Comment by ORESoftware** [comment]
@andy-ms thanks is there a Twitter account or Slack account that we can follow to get notified of new versions of `@types/x`?

**Comment by andy-ms** [comment]
@ORESoftware No, but you could run `npm outdated` in an npm package with `@types` dependencies to see if there are any updates.

**Comment by ORESoftware** [comment]
@andy-ms  got it thanks

**Pull request merged**